doc/manuals/user/ngsiv1autocast.md

```diff
@@ -0,0 +1,29 @@
+# NGSIv1 Autocast
+
+By default, due to parsing technology, NGSIv1 attribute creation/update operations always store attributes as strings,
+even when then input JSON uses other native type (such as Number, Boolean, etc.). In addition, the DateTime type is not
+supported in NGSIv1 so attributes of this type are stored as plain strings. This is a problem when advance functionality
+(such as NGSIv2 filtering) want to be used on those attribute values.
+
+The NGSIv1 autocast feature allows to alleviate this situation. It is activated with the `-ngsiv1Autocast`
+[CLI option](../admin/cli.md). When enabled, Orion Context Broker will interpret the following attribute types in
+NGSIv1 create/update operations:
+
+* "Number" (and its alias "Quantity")
+* "DateTime" (and its alias "ISO8601")
+* "Boolean"
+
+The processing is as follows:
+
+* At attribute creation time, if the attribute type _in the request_ is one of the above, a casting to the right
+  type is done before storing the attribute.
+* At attribute modification time, involving such modification _both attribute value and type_, if the attribute
+  type _in the request_ is one of the above, a casting to the right type is done before storing the attribute.
```

Comment:
right ... corresponding?

Reply:
Fixed in f65e5f29a1e53753c25d1c2e81ea7e3277fb241c